.github/workflows/closed-issue-message.yml

```
@@ -0,0 +1,17 @@
+name: Closed Issue Message
+on:
+  issues:
+    types: [closed]
+jobs:
+  auto_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aws-actions/closed-issue-message@v1
+        with:
+          # These inputs are both required
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          message: |
+            ### ⚠️COMMENT VISIBILITY WARNING⚠️
+            Comments on closed issues are hard for our team to see.
+            If you need more assistance, please either tag a team member or open a new issue that references this one.
```

Would it be possible to change this to:
```
If you need more assistance, please open a new issue that references this one.
```
?

I think we should encourage developers to open new issues to keep conversations on issues/bug reports/feature requests scoped to the topic at hand so that it's easier to reference at a later time without the additional conversations.

---

I agree and thank you for the feedback Danielle!

```suggestion
            If you need more assistance, please open a new issue that references this one.
```

---

This looks good to me 👍🏼